string Class
geometry_msgs/Pose pose
int32 detect_id